tests/common/platform/device_utils.py

```diff
@@ -18,6 +18,8 @@
 from tests.common.mellanox_data import is_mellanox_device
 from tests.common.platform.reboot_timing_constants import SERVICE_PATTERNS, OTHER_PATTERNS, SAIREDIS_PATTERNS, \
     OFFSET_ITEMS, TIME_SPAN_ITEMS, REQUIRED_PATTERNS
+from tests.common.platform.test_subnet_decap import build_encapsulated_vlan_subnet_packet, \
```

Comment:
Please move the test back to decap dir. Move the logic that is to be used (build_encapsulated_vlan_subnet_packet) by both decap and advanced_reboot case to a common location.